tests/integ/conftest.py

```diff
@@ -61,6 +61,7 @@ def connection(db_parameters):
         database=ret["database"],
         account=ret["account"],
         protocol=ret["protocol"],
+        role=ret.get("role")
```

Comment:
This probably fits a separate PR

Reply:
I'm just worried that in case this broke some stuff then we need to revert the meaty chunk of changes in this PR. Don't have strong feeling.

Reply:
I have merged this into main in a separate PR. If you pull changes from main, you will be good to go.

Reply:
ok, done